swig/CMakeLists.txt

```diff
@@ -0,0 +1,38 @@
+cmake_minimum_required(VERSION 3.5.0)
+
+include(ExternalProject)
+
+find_package(SWIG QUIET)
```

Comment:
You should add SWIG to the readme under required. Also it looks like I missed libccd

Reply:
updated